DataFormats/Detectors/TRD/include/DataFormatsTRD/RawData.h

```diff
@@ -218,6 +218,183 @@ struct TRDFeeID {
   };
 };
 
+/// \structure DigitHCHeader
+/// \brief Digit version of the TrackletHCHeader above, although contents are rather different.
+//  TODO come back and comment the fields or make the name more expressive, and fill in the jjjjjjj
+struct DigitHCHeader {
+  //
+  //             10987654321098765432109876543210
+  // uint32_t:   00000000000000000000000000000000
+  //
+  union { //page 109 section 15.6.1 in tdp
```

Comment:
Not sure I understand it. The DigitHCHeader is not described in the tdp, right?

Reply:
Its actually the only half chamber header defined in the TDP, and yip the citation is correct, although I see now the page number is actually 108, but that is likely to change, will take page number out.